packages/spindle-ui/src/Pagination/util/url.ts

```diff
@@ -0,0 +1,3 @@
+export function createUrl(url: string, pageNumber: string) {
```

Comment:
nits. 変換元が第2引数をnumberからstringに変換しているようなので、ここではnumberを受け取って関数内でstringに変換してもよさそうだなと思いました
```suggestion
export function createUrl(url: string, pageNumber: number) {
```